src/components/GoAccordion/GoCollapse.vue

```diff
@@ -24,7 +24,7 @@
         </div>
         <div class="go-header__title">
           <slot name="header">
-            <h5 v-html="item.title"></h5>
+            <h5 v-if="item" v-html="item.title"></h5>
```

Comment:
v-if="item.title" ?

Reply:
I want to test if "item" is null or undefined, more than if title is.

Reply:
But if item is not null or undefined, I think the tag go on.

Reply:
It does, I just tested it => this v-if is useless, i'll remove it.